audonnx/core/model.py

```diff
@@ -46,13 +47,31 @@ class Model(audobject.Object):
     to see all available methods.
 
     Args:
-        path: path to model file
+        path: ONNX model or path to model file
         labels: list of labels or dictionary with labels
         transform: callable object or a dictionary of callable objects
         device: set device
             (``'cpu'``, ``'cuda'``, or ``'cuda:<id>'``)
             or a (list of) provider(s)_
 
+    Example:
+        >>> import audonnx.testing
+        >>> model = audonnx.testing.create_model([[1, -1]])
```

Comment:
For the docstring example it would be nicer if we could instantiate from a folder using `model = Model(path)`

Reply:
Ok, we actually had such an example under `__call__()`, so I merged the two, see description.